src/containers/Desk.js

```diff
@@ -3,56 +3,126 @@ import NavBar from '../components/NavBar'
       .then(resp => resp.json())
       .then(json => {
-        this.setState({ letterStack: json })
+        this.setState({ 
+          letterStack: json,
+          plane: json[Math.round(Math.random() * 10)] //this is just for testing!
+        })
       })
   }
 
+  handleWriteClick = () => {
+    this.setState({
+      isWrite: true
+    })
+  }
+
   renderWrite = () => {
-    console.log('write')
-    return <Write />
+    return (
+      <Write accountId={this.props.accountId} handleCloseClick={this.handleCloseClick} />
+    )
   }
 
-  renderRead = e => {
-    console.log(e.target.id, this)
-    return <Read />
+  handlePlaneClick = (_e, letter) => {
+    this.setState({
+      isRead: true
+    })
+    this.incrementLetterViews(letter)
   }
 
-  renderPlanes = () => {
-    return this.state.letterStack.map(letter => {
-      return <Plane key={letter.id} letter={letter} handleClick={this.renderRead} />
+  incrementLetterViews = (letter) => {
+    fetch(LETTERS_URL + `/${letter.id}`, {
+      method: 'PATCH',
+      headers: {
+        'Content-Type': 'application/json',
+        'Accept': 'application/json'
+      },
+      body: JSON.stringify({
+        num_views: letter.num_views + 1 
+      })
     })
+    .then(res => res.json())
+    .then(console.log)
   }
 
+  renderRead = (letter) => {
+
+    return (
+      <Read letter={letter} handleWriteClick={this.handleWriteClick} />
+    )
+  }
+
+  renderCreateResponse = (letter) => {
+    return (
+      <CreateResponse accountId={this.props.accountId} letter={letter} isRead={true} isWrite={true} handleCloseClick={this.handleCloseClick}/>
+    )
+  }
+
+  handleJournalClick = () => {
+    this.setState({
+      isJournal: true
+    })
+  }
+
+  renderJournal = () => {
+    return (
+      <Journal />
+    )
+  }
+
+  handleCloseClick = () => {
+    this.setState({
+      isWrite: false,
+      isRead: false,
+      isJournal: false,
+      plane: this.state.letterStack[Math.round(Math.random() * 10)] //for testing purposes
+    })
+  }
+
+  
+  isEmptyDesk = () => {
+    const { isWrite, isRead, isJournal } = this.state
+    return !isWrite && !isRead && !isJournal
+  }
+
+
   render() {
+  
+    const { plane, isWrite, isRead, isJournal } = this.state
+
     return (
       <>
-        <NavBar handleWrite={this.renderWrite} handleRead={this.renderRead} />
-        {this.renderPlanes()}
-        <Write />
-        <Read />
+        <NavBar isWrite={isWrite} isRead={isRead} isJournal={isJournal} handleWriteClick={this.handleWriteClick} handleJournalClick={this.handleJournalClick} handleCloseClick={this.handleCloseClick} /> 
+        { isWrite && !isRead ? this.renderWrite() : null }
+        { isRead && !isWrite ? this.renderRead(plane) : null }
+        { isWrite && isRead ? this.renderCreateResponse(plane) : null }
+        { isJournal ? this.renderJournal() : null }
+        { plane && this.isEmptyDesk() ? <Plane handlePlaneClick={this.handlePlaneClick} plane={plane}/> : null } 
```

Comment:
This logic will probably change a bit once I finish up my part, but the isEmptyDesk method will be super helpful